### ISSUE TITLE
 [MdInput] Zooming when focusing input-fields on iOS

### ISSUE THREAD

**Issue by Bolandish** [vuematerial/vue-material#789]
Hi! 

When focusing input-fields on iOS the device zooms into the field. 
See my gif.

![jun -12-2017 14-40-31](https://user-images.githubusercontent.com/7235443/27071453-57d87496-501d-11e7-912a-ed6c55562dae.gif)

**Comment by pablohpsilva** [comment]
Hello! Thank you for opening this issue.

Could you please provide a gif instead of a png? I just want to make sure before replying what I think the issue can be.

Thanks!

**Comment by Bolandish** [comment]
Ah, Sorry! I Dont know what happened :) 
![jun -12-2017 14-40-31](https://user-images.githubusercontent.com/7235443/27034279-30ecba9a-4f7d-11e7-9f78-ea4e2a393462.gif)

**Comment by Samuell1** [comment]
Closing this issue as our focus is on the new 1.0.0 version.

**Comment by Samuell1** [comment]
Its not issue on our side its because iOS have autozoom for inputs. @qubitron

**Comment by qubitron** [comment]
@Samuell1 From the hint in #417 I was able to figure out this is happening because the md-input has an initial font size of 1px, causing the auto-zoom to zoom in way too far when you tap on the input control.

I was able to fix this issue by putting the following style in my App.vue:
```css
.md-input {
  font-size: 16px !important;
}
```

It seems like this could be a simple fix to change the font-size to 16px. Without it, the the vue-material framework is unusable for me on mobile devices, and with this change it works great.
